.github/workflows/heroku.yaml

```diff
@@ -0,0 +1,54 @@
+# Your workflow name.
+name: Deploy to Heroku.
+
+# Run workflow on every push to master branch.
+on:
+  push:
+    branches:
+      - master
```

Comment:
Although this will only run on the master branch, it's worth considering whether we'd want to ensure that it's only running after the tests have run - e.g. if someone had to deal with merge conflicts and then pushed to master, we'd want the tests on the new master _before_ we potentially deploy 

That can be quite fiddly with this structure unfortunately - the easiest solution would likely to be to fold this back into the `ci.yaml` file so that you can easily gate this on the earlier job